src/ColumnSelectionStep/ColumnSelectionStep.tsx

```diff
@@ -49,11 +51,19 @@ function minBinSize(columnType: ColumnType) {
   return columnType === 'real' ? MIN_BIN_SIZE_REAL : MIN_BIN_SIZE_INTEGER;
 }
 
+function numericGeneralizationTooltip(binSize: number, binStart: number) {
+  return binStart.toString() + '-' + (binStart + binSize).toString();
```

Comment:
It is not clear from this that the upper bound is non-inclusive. Maybe `[0; 1)` is better?